Python/rerf/RerF.py

```diff
@@ -17,6 +17,12 @@ def fastRerF(
     mtryMult=1.5,
     fractionOfFeaturesToTest=None,
     seed=None,
+    imageHeight=0,
+    imageWidth=0,
+    patchHeightMax=0,
+    patchHeightMin=0,
+    patchWidthMax=0,
```

Comment:
should these default to 0?

Reply:
We have no way of knowing what they should be and 0 will yield a runtime error with a message.

Reply:
patch height/widths could have intelligent defaults...  also fine with leaving this as is since we're likely going to deprecate this submodule